test_phage_reference/nbconverted/1_align_quantify_samples.py

```diff
@@ -0,0 +1,211 @@
+
+# coding: utf-8
+
+# # Aign and quantify samples
```

Comment:
You probably mean "align". Feel free to slap me next time you see me for pointing out typos this minor.